README.md

```diff
@@ -10,52 +10,94 @@
+    using_bill: false # disable if you don't have bills or bill payments in QuickBooks
+    using_credit_memo: false # disable if you don't have credit memos in QuickBooks
+    using_department: false # disable if you don't have departments in QuickBooks
+    using_deposit: false # disable if you don't have deposits in QuickBooks
+    using_estimate: false # disable if you don't have estimates in QuickBooks
+    using_invoice: false # disable if you don't have estimates in QuickBooks
+    using_invoice_bundle: false # disable if you don't have estimates in QuickBooks
+    using_journal_entry: false # disable if you don't have estimates in QuickBooks
+    using_payment: false # disable if you don't have estimates in QuickBooks
+    using_refund_receipt: false # disable if you don't have estimates in QuickBooks
+    using_transfer: false # disable if you don't have estimates in QuickBooks
+    using_vendor_credit: false # disable if you don't have estimates in QuickBooks
+    using_sales_receipt: false # disable if you don't have estimates in QuickBooks
+    using_purchase_order: true # enable if you want to include purchase orders in your staging models
+    using_credit_card_payment_txn: true # enable if you want to include credit card payment transactions in your staging models
 ```
+</details>
+
+## (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™
+<details><summary>Expand for details</summary>
+<br>
+
+Fivetran offers the ability for you to orchestrate your dbt project through [Fivetran Transformations for dbt Core™](https://fivetran.com/docs/transformations/dbt). Learn how to set up your project for orchestration through Fivetran in our [Transformations for dbt Core setup guides](https://fivetran.com/docs/transformations/dbt#setupguide).
+</details>
+
 
-### Union Multiple Quickbooks Connectors
-If you have multiple Quickbooks connectors in Fivetran and would like to use this package on all of them simultaneously, we have provided functionality to do so. The package will union all of the data together and pass the unioned table into the transformations. You will be able to see which source it came from in the `source_relation` column of each model. To use this functionality, you will need to set either the `quickbooks_union_schemas` or `quickbooks_union_databases` variables:
+### Union Multiple QuickBooks Connectors
+<details><summary>Expand for details</summary>
+If you have multiple QuickBooks connectors in Fivetran and would like to use this package on all of them simultaneously, we have provided functionality to do so. The package will union all of the data together and pass the unioned table into the transformations. You will be able to see which source it came from in the `source_relation` column of each model. To use this functionality, you will need to set either the `quickbooks_union_schemas` or `quickbooks_union_databases` variables:
```

Comment:
This should be it's own `(Optional) Step 5: Additional Configurations`. See [here](https://github.com/fivetran/dbt_twitter_organic#optional-step-4-additional-configurations) as an example.

Reply:
Can you follow the same structure that we have in Twitter Organic as to what is included in the additional configuration. For example the following should be included in that section:
- Unioning feature
- Change the build schema
- Change source table ref